### DISCUSSION TITLE
Impossible to run locally... What should I do?

### DISCUSSION THREAD

**Discussion by drfcozapata** [nuxt/nuxt#17017] [Questions]
I forked the repo and clone it because want to translate the docs to spanish.
Almost every dependency report some kind of error when try to run "npx yarn install"
Tried everything, but I can't installed it.
Please, ckeck it out. Thanks!
(and sorry my spanglish)

**Comment by danielroe** [comment]
Check out https://v3.nuxtjs.org/community/framework-contribution#set-up-your-local-development-environment

**Comment by drfcozapata** [comment]
Thanks Daniel!
Blessings


[image: Dr. Francisco Zapata] <https://linkedin.com/in/drfcozapata> *Francisco
Zapata*

Freelancer | Web Developer Jr | Editorial Designer | Writer | Pediatrician
<https://twitter.com/DrFcoZapata>  <https://t.me/drfcozapata>
<https://www.youtube.com/channel/UCRLoYe7kEUcpt18b0pTRQ9Q>
<https://linkedin.com/in/drfcozapata>  <https://github.com/drfcozapata>
+58 412 0918548Puerto Ordaz, Bol, Venezuela
